.circleci/bump-version.js

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/node
+
+const { Octokit } = require('@octokit/rest')
+const { createAppAuth } = require('@octokit/auth-app')
+const fs = require('fs').promises
+
+const bumpVersion = async () => {
+  try {
+    const versionPath = 'version.json'
+
+    const versionFile = await fs.readFile(versionPath)
+    const { versionName, versionCode } = JSON.parse(versionFile)
+    const versionNameParts = versionName.split('.')
+
+    const date = new Date()
+    const year = date.getFullYear()
+    const month = date.getMonth() + 1
+
+    const versionNameCounter = year === versionNameParts[0] && month === versionNameParts[1] ? versionNameParts[2] + 1 : 0
+    const newVersionName = `${year}.${month}.${versionNameCounter}`
+    const newVersionCode = versionCode ? versionCode + 1 : undefined
```

Comment:
When is the versionCode undefined? If it is undefined it will always stay undefined, right?

Reply:
This is for the react-native app which will also use this. But yeah right, for the webapp it will stay undefined. I'll add a comment.